content/templates/pages/graphql_server.md

```diff
@@ -0,0 +1,5 @@
+---
+hidden: true
+---
+
+{{< template-page "graphql_server" >}}
```

Comment:
are we removing the content on these pages? the graphql server, rustwasm (which actually appears to be missing here), and router boilerplates have markdown content that has been removed during this migration

Reply:
The one link it stil there. I will make this part of the work in https://github.com/cloudflare/workers-docs/issues/535
To move whatever markdown was in `templates/*/*.md` to `templates/pages/*.md`